### DISCUSSION TITLE
How do you manage reports from the GUI?

### DISCUSSION THREAD

**Discussion by b-u-g-s** [Pithus/bazaar#24] [Q&A]
Hi,

I managed to logon to my local instance thanks to @christalib 
Now that I see the "admin" menu I am just wondering how do you manage reports? by that I mean, I am logged in as a super admin but I cannot see the reports I generated without an ID, so how can I "manage" them. i.e.: choose which one I want to delete! 

Also, I don't see a menu to show me all the reports/apk available.
I can achieve that by just typing "handle: *" in the search menu, but shouldn't there be a main dashboard listing all the reports available? and in turns an option to delete those reports :)

**Comment by U039b** [comment]
Hi!
There is no way to manage (edit, delete) reports, Pithus works in an append-only way by design. To list the 50 latest reports, you can search for `*` in the search field. Pithus does not list all reports to reduce the workload on the server. I think you should plan to add this feature.
The only thing you can do regarding the reports is recomputing one or all of them using a [dedicated command](https://github.com/Pithus/bazaar/blob/master/bazaar/core/management/commands/update_reports.py).

```
docker-compose -f local.yml run --rm django python manage.py update_report "[sha256|*]" [one-letter task identifier]
```
You can update a single report by specifying its `sha256` or update all reports by specifying `*`. To choose the analysis step to rerun, specify one or more one-letter identifier. As an example, specifying `yf` will rerun the Yara analysis and the frosting one. 

Example:
```
docker-compose -f local.yml run --rm django python manage.py update_report "*" yf
```

**Reply by b-u-g-s** [reply]
ok thanks for that!
